src/test/java/com/booster/server/VocabularyEntryControllerTest.java

```diff
@@ -0,0 +1,50 @@
+package com.booster.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.assertj.core.groups.Tuple;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+class VocabularyEntryControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+    @Autowired
+    ObjectMapper objectMapper;
+    @Autowired
+    VocabularyEntryRepository vocabularyEntryRepository;
+
+    @Test
+    void create() throws Exception {
+        assertThat(vocabularyEntryRepository.findAll()).isEmpty();
+
+        mockMvc.perform(MockMvcRequestBuilders.post("/vocabulary-entry")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content(objectMapper.writeValueAsString(CreateVocabularyEntryInput.builder()
+                                .name("a")
+                                .description("d")
+                                .build())))
```

Comment:
Let`s create a test with a case of business logic, not just with random values.

Reply:
Fixed